doc/classes/Particles.xml

```diff
@@ -266,36 +271,47 @@
 		<member name="draw_passes" type="int" setter="set_draw_passes" getter="get_draw_passes">
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+			If [code]true[/code] particles are being emitted. Default value: [code]true[/code].
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+			Time ratio between each emission. If [code]0[/code] particles are emitted continuously. If [code]1[/code] all particles are emitted simultaneously. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
+			Amount of time each particle will exist. Default value: [code]1[/code].
 		</member>
 		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
+			If [code]true[/code] particles will use the parent node's coordinate space. If [code]false[/code] they will use global coordinates. Default value: [code]true[/code].
```

Comment:
Suggestion:
`If [code]true[/code] particles use the parent node's coordinate space. If [code]false[/code] they use global coordinates. Default value: [code]true[/code].`